### PR TITLE
BULK VALIDATION description-missing

### PR DESCRIPTION
**Pull request by mSarts**
## Summary

Fixes 30 issues across 30 files.

Fixes no issue numbers.

### CODE REVIEW

File: dotnet-desktop-guide/framework/wpf/controls/expander-styles-and-templates.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Expander Styles and Templates"
+description: Learn about the styles and templates for te Expander control. You can modify the default ControlTemplate to give the control a unique appearance.
```
Comment (JustPies) [inline dotnet-desktop-guide/framework/wpf/controls/expander-styles-and-templates.md:3]: for "the" Expander control.

File: dotnet-desktop-guide/framework/wpf/controls/how-to-create-a-control-that-has-an-access-key-and-text-wrapping.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "How to: Create a Control That Has an Access Key and Text Wrapping"
+description: Learn how to create a control that has an access key and text wrapping by means of the included code example in XAML.
```
Comment (JustPies) [inline dotnet-desktop-guide/framework/wpf/controls/how-to-create-a-control-that-has-an-access-key-and-text-wrapping.md:3]: I think you should put a comma after "wrapping" just so it has the same punctuation/construction as your other descriptions.

File: dotnet-desktop-guide/framework/wpf/controls/expander-styles-and-templates.md (hunk 1 above)
Comment (mSarts) [reply dotnet-desktop-guide/framework/wpf/controls/expander-styles-and-templates.md]: Addressed.

File: dotnet-desktop-guide/framework/wpf/controls/how-to-create-a-control-that-has-an-access-key-and-text-wrapping.md (hunk 2 above)
Comment (mSarts) [reply dotnet-desktop-guide/framework/wpf/controls/how-to-create-a-control-that-has-an-access-key-and-text-wrapping.md]: Addressed.

**Comment by mSarts** [comment]
**Global effort to fix validation errors**

 @adegeo   

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

File: dotnet-desktop-guide/framework/wpf/controls/grid-how-to-topics.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Grid How-to Topics"
+description: Learn about how to position elements using the Grid element via the collected lists of helpful links in this article.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/grid-how-to-topics.md:3]:
```suggestion
description: Learn about how to position elements using the Grid layout control via a curated list of helpful links in this article.
```

File: dotnet-desktop-guide/framework/wpf/controls/grid.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Grid"
+description: Learn about the Grid element, which is used to precisely position content in rows and columns via the collected list of helpful links.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/grid.md:3]:
```suggestion
description: Learn about the Grid layout control, which is used to precisely position content in rows and columns via the collected list of helpful links.
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**